packages/types/src/tax/common.ts

```diff
@@ -22,6 +22,18 @@ export interface TaxRateDTO {
    * Holds custom data in key-value pairs.
    */
   metadata: Record<string, unknown> | null
+  /**
+   * The id of the Tax Region the rate is associated with.
+   */
+  tax_region_id: string
+  /**
+   * Flag to indicate if the Tax Rate should be combined with parent rates.
+   */
+  is_combinable: boolean
```

Comment:
I am completely clueless on this one, what does it mean to be able to combine a tax rate with a parent rate? If this is explained somewhere in Notion feel free to just share a link to that 😄

Reply:
In some countries, multiple authorities can charge tax. For example, in Canada, you pay a federal sales tax and a provincial sales tax. 

The way that would be set up would be something like:
- COUNTRY Region
  - Tax Rate: X
  - PROVINCE Region
    - parent_id: state region id
    - Tax Rate: Y

The is_combinable flag indicates whether the end customer should be charged X+Y or just Y. 

Does this make more sense?

Reply:
Yeah makes perfect sense, thanks @srindom !